lib/completion.ts

```diff
@@ -258,7 +258,9 @@ export class Completion implements CompletionInstance {
     if (!this.zshShell) {
       completions.push(dashes + key);
     } else {
-      const desc = descs[key] || '';
+      const aliasKey = this?.aliases?.[key]?.[0];
```

Comment:
what about cases where there are multiple aliases associated with a key? should this be a loop?

Reply:
Oh, Thank you. Yes, sometimes the `this?.aliases?.[key]?.[0]` will be another alias without a description.
I will make it a loop.